WIP-CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Fixed a templating error which occurs when more than one workload cluster is specified under the `global.clustersMonitoring` in the `sc-config.yaml`
 - Fix `KubeletDown` alert rule, did previously not alert if a kubelet was missing.
 - Add permissions to the `alerting_full_access` role in Opensearch to be able to view notification channels.
+- Added a fluentd <exclude> pattern to our ingress-nginx-controller-containers to prevent them from overloading and `fluent-plugin-record-modifier` was added to our image `ghcr.io/elastisys/fluentd:v3.4.0-ck8s5` to prevent further errors from happening.
```

Comment:
The `<exclude>` thing doesn't apply any more right?

Reply:
Yeah it shouldn't be needed if the new plugin fixes it all :thinking_face: